Sources/PIRService/ReloadService.swift

```diff
@@ -71,17 +76,18 @@ actor ReloadService: Service {
         let config = try JSONDecoder().decode(ServerConfiguration.self, from: configData)
 
         var allowedUsers: [String: UserTier] = [:]
-        for (tier, users) in config.users {
-            for user in users {
-                if let existingTier = allowedUsers[user],
+        for usergroup in config.users {
```

Comment:
```suggestion
        for userGroup in config.users {
```

Or maybe just `group`.